apis/integreatly/v1alpha1/types/types.go

```diff
@@ -44,11 +44,12 @@ func (sm StatusMessage) WrapError(err error) StatusMessage {
 
 // +kubebuilder:object:generate=true
 type ResourceTypeStatus struct {
-	Strategy  string        `json:"strategy,omitempty"`
-	Provider  string        `json:"provider,omitempty"`
-	SecretRef *SecretRef    `json:"secretRef,omitempty"`
-	Phase     StatusPhase   `json:"phase,omitempty"`
-	Message   StatusMessage `json:"message,omitempty"`
+	Strategy       string        `json:"strategy,omitempty"`
+	Provider       string        `json:"provider,omitempty"`
+	VersionDetails string        `json:"versionDetails,omitempty"`
```

Comment:
Naming Suggestion. wdyt about using `version` instead of `versionDetails`? Or if we think there could be other items we want to expose later, we could add a `config` type which contains `version` which could be extended at a later date? Also in terms of future proofing I think we're fairly safe to add a version. My guess is that any cloud provider would expose a version of the postgres at least but we should keep in mind that this Status applies to any and all resource objects managed by cro. 👍

Reply:
I was aiming for generic, but would say that `version` is fairly universal no problem whit that change.